package.json

```diff
@@ -158,6 +156,8 @@
     "babel-jest": "22.1.0",
     "chai": "3.4.1",
     "enzyme": "3.2.0",
+    "enzyme-adapter-react-16": "npm:enzyme-react-adapter-future",
```

Comment:
For some reason they haven't published the fixes yet, but I think this is fine for now if tests are happy :/